rfremoterunner/executor_argparser.py

```diff
@@ -0,0 +1,149 @@
+                                 ' it can also be a simple pattern where `*` matches anything and `?` matches any '
+                                 'char. To specify multiple, concatenate with a colon. Example: -t Foo*:Bar*')
+        parser.add_argument('-i', '--include',
+                            help='Select test cases to run by tag. Similarly as name  with --test, tag is case and '
+                                 'space insensitive and it is possible to use patterns with `*` and `?` as wildcards. '
+                                 'Tags and patterns can also be combined together with `AND`, `OR`, and `NOT` '
+                                 'operators. Examples: --include foo, --include bar*, --include fooANDbar*')
+        parser.add_argument('-e', '--exclude',
+                            help='Select test cases not to run by tag. These tests are not run even if included with '
+                                 '--include. Tags are matched using the rules explained with --include.')
+
+        parser.add_argument('-L', '--loglevel',
+                            help=' Threshold level for logging. Available levels: TRACE, DEBUG, INFO (default), WARN, '
+                                 'NONE (no logging). Use syntax `LOGLEVEL:DEFAULT` to define the default visible log '
+                                 'level in log files. Examples: --loglevel DEBUG --loglevel DEBUG:INFO')
+        return parser
+
+    def get_log_html_output_location(self):
+        """
+        Determine the local output file location of the log.html based on the input arguments
+
+        :return: Path to where the log html file should be saved
+        :rtype: str
+        """
+        return self._resolve_output_path(self.log, 'remote_log.html')
+
+    def get_output_xml_output_location(self):
+        """
+        Determine the local output file location of the output.xml based on the input arguments
+
+        :return: Path to where the output xml file should be saved
+        :rtype: str
+        """
+        return self._resolve_output_path(self.output, 'remote_output.xml')
+
+    def get_report_html_output_location(self):
+        """
+        Determine the local output file location of the report.html based on the input arguments
+
+        :return: Path to where the report html file should be saved
+        :rtype: str
+        """
+        return self._resolve_output_path(self.report, 'remote_report.html')
+
+    def _resolve_output_path(self, argument_val, filename):
+        """
+        Determine a path to output a file artifact based on whether the user specified the specific path
+
+        :param argument_val: Value of the specific argument e.g. from --output
+        :type argument_val: str
+        :param filename: Name of the file e.g. log.html
+        :type filename: str
+
+        :return: Absolute path of where to save the test artifact
+        :rtype: str
+        """
+        parent_dir = self.outputdir or '.'

```

Comment:
you wouldnt have to do this if outputdir was initialised to '.'. this code also exists in executor.py

Reply:
set the default to '.' when defining the argument in the ArgumentParser